src/components/Article-preview.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from "react";
+import { fetchArticles } from "../utils/api-fetch-articles";
+
+const ArticlePreview = () => {
```

Comment:
i would argue this is the same as your al articles component - you can use the same component and just limit the number of articles shown (although i would say you don't need this at all - you can have your all articles on your home page)